### ISSUE TITLE
Integrate multiple proxies per voldrv

### ISSUE THREAD

**Issue by wimpers** [openvstorage/framework#1274]
The framework should setup 8 proxies per voldrv.

**Comment by khenderick** [comment]
```
    ...
    "backend_connection_manager": {
        "backend_type": "MULTI", 
        "1": {
            "backend_interface_retries_on_error": 5, 
            "alba_connection_preset": "default", 
            "backend_type": "ALBA", 
            "alba_connection_transport": "TCP", 
            "alba_connection_timeout": 15, 
            "backend_interface_retry_backoff_multiplier": 2.0, 
            "backend_interface_retry_interval_secs": 1, 
            "alba_connection_host": "10.100.169.100", 
            "alba_connection_port": 26211
        }, 
        "0": {
            "backend_interface_retries_on_error": 5, 
            "alba_connection_preset": "default", 
            "backend_type": "ALBA", 
            "alba_connection_transport": "TCP", 
            "alba_connection_timeout": 15, 
            "backend_interface_retry_backoff_multiplier": 2.0, 
            "backend_interface_retry_interval_secs": 1, 
            "alba_connection_host": "10.100.169.100", 
            "alba_connection_port": 26209
        }, 
        ...
    },
    ...
```

**Comment by khenderick** [comment]
Fixed by #1276, packaged in openvstorage-2.7.9-rev.4457.15529c2
